internal/commands/app/diff_test.go

```diff
@@ -292,4 +342,19 @@ func TestAppDiffInputs(t *testing.T) {
 			tc.test(t, tc.inputs, profile)
 		})
 	}
+
+	t.Run("should return error with include node modules and include package json are both set", func(t *testing.T) {
```

Comment:
ok, i'm officially confused in this file.  this _is_ the correct test I thought was missing, but we also want this to have coverage for when `--include-dependencies` is set (make that a test table too, it will be more obvious they're essentially testing the same thing)

Reply:
Yeah, the part above is `TestDiff` and this part is `TestDiffInput`, I will delete this test and modify the test above so it does the `Resolve` testing.